src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -931,6 +936,26 @@ private async Task RegisterSubscriptionUpdateAction(
         string updateMessage = subscriptionUpdateAction.ToString();
         await _sqlClient.RegisterSubscriptionUpdate(subscriptionId, updateMessage);
     }
+    private static PullRequestUpdateSummary CreatePrSummaryFromInProgressPr(
+        InProgressPullRequest pr,
+        string targetRepo)
+    {
+        return new PullRequestUpdateSummary(
+            pr.Url,
+            pr.CoherencyCheckSuccessful,
+            pr.CoherencyErrors,
+            pr.RequiredUpdates,
+            pr.ContainedSubscriptionUpdates.Select(su => new SubscriptionUpdateSummary
+            {
+                SubscriptionId = su.SubscriptionId,
+                BuildId = su.BuildId,
+                SourceRepo = su.SourceRepo,
+                SourceSHA = su.SourceSHA
+            }).ToList(),
```

Comment:
What's the point of this transformation - it does not add any new data? Can't we just keep the type from before?

Reply:
Ah, I see, it's in another project

Reply:
However, `TargetRepo` is defined on `SubscriptionUpdateSummary` and it's not set here. But it's already in `PullRequestUpdateSummary` so maybe it's not needed there?

Reply:
How about the fact that we can get the target repo from the `IPullRequest.Url`? Can we take advantage of that and not add a new field? Maybe we won't need any changes to the model? Or was there another field added?